pkg/oauth/server/osinserver/registrystorage/storage.go

```diff
@@ -248,6 +255,20 @@ func (s *storage) convertToAccessToken(data *osin.AccessData) (*oauthapi.OAuthAc
 	if err := s.user.ConvertToAccessToken(data.UserData, token); err != nil {
 		return nil, err
 	}
+
+	// Check if the optional inactivity Timeout needs to be set
+	var timeoutsin int32 = 0
+	wrapper := data.Client.(*clientWrapper)
+	inactivityTimeout := wrapper.GetTokenTimeoutSeconds()
```

Comment:
This block would get simpler if you made `storage.tokentimeout` an `int32` that was defaulted to `0` in `New`.

Reply:
I'll look into that.

Reply:
done